components/widgets/menu/menu_items/menu_item_link.tsx

```diff
@@ -9,9 +10,22 @@ import menuItem from './menu_item';
 type Props = {
     to: string;
     text: React.ReactNode;
+    className?: string;
+    disabled?: boolean;
+    sibling?: React.ReactNode;
 }
 
-export const MenuItemLinkImpl: React.FC<Props> = ({to, text}: Props) => <Link to={to}><span className='MenuItem__primary-text'>{text}</span></Link>;
+export const MenuItemLinkImpl: React.FC<Props> = ({to, text, className, disabled, sibling}: Props) => (
```

Comment:
if this component doesn't receive/use children, please, remove the React.FC type.